ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/AbstractConnectorExecutor.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.algorithms.remote;
+
+import org.opensearch.ml.common.connector.Connector;
+import org.opensearch.ml.common.connector.ConnectorHttpClientConfig;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public abstract class AbstractConnectorExecutor implements RemoteConnectorExecutor {
+    private ConnectorHttpClientConfig httpClientConfig;
+
+    public void validate(Connector connector) {
```

Comment:
should we rename this to "initialization"? No Exception is handled or thrown so this function looks more like an initialization rather than validation.

Reply:
addressed.